src/en/charms-storage.md

```diff
@@ -415,6 +430,72 @@ provider currently supports a single pool configuration attribute:
 
 ### LXD (lxd)
 
+LXD-based models have access to the 'lxd' storage provider. The LXD provider
+does not currently have any specific configuration options.
```

Comment:
> Why expose driver when it's integral to lxd-pool? IIUC, doing so would allow for incompatible pairs (e.g. btrfs driver with a zfs-based pool)?

Juju creates the LXD storage pools if they don't exist. If they *do* exist, Juju will ensure that they have the same attributes as specified by the Juju user.

> Anyway, I don't know how to pass these configuration options. Can you give a few explicit examples, including one where options are passed to LXD?

This is done by creating a Juju storage pool. e.g.

```go
juju create-storage-pool lxd-zfs lxd driver=zfs lxd-pool=juju-zfs zfs.pool_name=juju-lxd
```